lib/integration_whitelist.rb

```diff
@@ -0,0 +1,113 @@
+INTEGRATION_WHITELIST = [
+  'active_directory',
+  'activemq',
+  'activemq_xml',
+  'agent_metrics',
+  'apache',
+  'aspdotnet',
+  'btrfs',
+  'cacti',
+  'cassandra',
+  'cassandra_nodetool',
+  'ceph',
+  'cisco_aci',
+  'cockroachdb',
+  'consul',
+  'coredns',
+  'couch',
+  'couchbase',
+  'crio',
+  # 'datadog_checks_base',   namespacing package for wheels (NOT AN INTEGRATION)
+  # 'datadog_checks_dev',      developer tooling for working on integrations (NOT AN INTEGRATION)
+  'directory',
+  'disk',
+  'dns_check',
+  'docker_daemon',
+  'dotnetclr',
+  'ecs_fargate',
+  'elastic',
+  'envoy',
+  'etcd',
+  'exchange_server',
+  'fluentd',
+  'gearmand',
+  'gitlab',
+  'gitlab_runner',
+  'go-metro',
+  'go_expvar',
+  'gunicorn',
+  'haproxy',
+  'hdfs_datanode',
+  'hdfs_namenode',
+  'http_check',
+  'hyperv',
+  # 'ibm_mq',        only supported on agent 6 because of binary dependencies
+  'iis',
+  'istio',
+  'kafka',
+  'kafka_consumer',
+  'kong',
+  'kube_controller_manager',
+  'kube_dns',
+  'kube_proxy',
+  'kubelet',
+  'kubernetes',
+  'kubernetes_state',
+  'kyototycoon',
+  'lighttpd',
+  'linkerd',
+  'linux_proc_extras',
+  'mapreduce',
+  'marathon',
+  'mcache',
+  'mesos_master',
+  'mesos_slave',
+  'mongo',
+  'mysql',
+  'nagios',
+  'network',
+  'nfsstat',
+  'nginx',
+  'nginx_ingress_controller',
+  'ntp',
+  'openldap',
+  'openmetrics',
+  'openstack',
+  'openstack_controller',
```

Comment:
Could we remove `openstack_controller`. I believe it never was included at this point, was it?

Reply:
Good catch. It was included because it was in the 6.9 branch, but I forgot to remove it since it was in the original blacklist.